android/scaffold/src/main/java/com/mobify/astro/scaffold/plugins/DoubleIconsPlugin.java

```diff
@@ -0,0 +1,149 @@
+package com.mobify.astro.scaffold.plugins;
+
+import com.mobify.astro.AstroActivity;
+import com.mobify.astro.AstroPlugin;
+import com.mobify.astro.PluginResolver;
+
+import android.content.Context;
+import android.util.TypedValue;
+import android.view.Gravity;
+import android.view.View;
+import android.widget.FrameLayout;
+import android.widget.LinearLayout;
+
+import com.mobify.astro.messaging.annotations.RpcMethod;
+import com.mobify.astro.plugins.headerbarplugin.HeaderContentItem;
+import com.mobify.astro.scaffold.R;
+
+public class DoubleIconsPlugin extends AstroPlugin {
+
+    public class DoubleIconItemView extends FrameLayout {
+        View view = null;
+        int contentGravity;
+        String id = "";
+
+        public DoubleIconItemView(Context context, int gravity) {
+            super(context);
+            this.contentGravity = gravity;
+        }
+
+        private void setPlugin(AstroPlugin plugin, String id) {
+            if (view != plugin.getView()) {
+                this.removeAllViews();
+
+                FrameLayout.LayoutParams layoutParams = new FrameLayout.LayoutParams(
+                        LayoutParams.WRAP_CONTENT,
+                        LayoutParams.WRAP_CONTENT
+                );
+                layoutParams.gravity = contentGravity;
+
+                this.addView(plugin.getView(), layoutParams);
+                view = plugin.getView();
+                this.id = id;
+            }
+
+            this.setVisibility(View.VISIBLE);
+        }
+    }
+
+    LinearLayout containerView;
+    DoubleIconItemView leftIcon;
+    DoubleIconItemView rightIcon;
+
+    class DoubleIconsItemClickListener implements View.OnClickListener {
+        DoubleIconItemView doubleIconItemView;
+
+        DoubleIconsItemClickListener(DoubleIconItemView doubleIconItemView) {
+            this.doubleIconItemView = doubleIconItemView;
+        }
+
+        @Override
+        public void onClick(View v) {
+            triggerEvent(doubleIconItemView.id, null);
+        }
+    }
+
+    public static int getActionBarHeight(AstroActivity activity) {
```

Comment:
This can be removed and we can now use HeaderUtilities.getActionBarHeight(activity) :)